bazel_codegen/lib/src/resolver/build_asset_uri_resolver.dart

```diff
@@ -28,6 +28,9 @@ class BuildAssetUriResolver implements UriResolver {
     }
   }
 
+  @override
+  void clearCache() {}
```

Comment:
Or extends? Then you can skip the blank override...

Reply:
> Or extends? Then you can skip the blank override...

Done.  PTAL